ClientApp/KachnaOnline/src/app/home/repeating-states/repeating-states.component.html

```diff
@@ -3,11 +3,11 @@ <h2>Pravidelná otevírací doba</h2>
 </div>
 
 <div class="d-flex justify-content-center border-bottom pb-2 mb-2">
-  <h5 *ngIf="hasRegularOpenings == undefined" class="text-muted">
+  <h5 *ngIf="hasRegularOpenings == undefined" class="text-muted text-center ">
```

Comment:
Why? The text is centered by its parent flexbox

Reply:
No, it is aligned to the left for the hardcoded messages.